src/amo/components/LandingPage.scss

```diff
@@ -1,5 +1,44 @@
 @import "~amo/css/inc/vars";
+@import "~core/css/inc/mixins";
 
 .LandingPage {
   padding: $padding-page;
 }
+
+.LandingPage-Header {
+  margin: 10px;
+  max-width: 100%;
+  text-align: center;
+}
+
+.LandingPage-Header-Text {
+  display: inline-block;
+  @include margin-start(20px);
+  @include text-align-start();
+  width: 250px;
+}
+
+.LandingPage-Heading {
+  font-size: 20px;
+}
+
+.LandingPage-Heading-Content {
+  margin-top: 5px;
+}
+
+Link--light {
```

Comment:
This is missing the `.` before `Link--light`. This should be moved into the styles for `Link` too.